### PR TITLE
 set default value for auto pause and auto scale settings of New-AzSynapseSparkPool cmdlet

### PR DESCRIPTION
**Pull request by yanjungao718**
## Description

 Set default value for auto pause and auto scale settings of New-AzSynapseSparkPool cmdlet 

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Synapse/Synapse/Commands/ManagementCommands/SparkPool/NewAzureSynapseSparkPool.cs [hunk 1]
```diff
@@ -178,13 +178,13 @@ public override void ExecuteCmdlet()
                 NodeCount = this.enableAutoScale ? (int?) null : this.NodeCount,
                 NodeSizeFamily = NodeSizeFamily.MemoryOptimized,
                 NodeSize = NodeSize,
-                AutoScale = !this.enableAutoScale ? null : new AutoScaleProperties
+                AutoScale = !this.enableAutoScale ? new AutoScaleProperties { Enabled = this.enableAutoScale } : new AutoScaleProperties
```
Comment (idear1203) [inline src/Synapse/Synapse/Commands/ManagementCommands/SparkPool/NewAzureSynapseSparkPool.cs:181]: >this.enableAutoScale [](http://example.com/codeflow?start=88&length=20)

Just use `false` for simplicity

File: src/Synapse/Synapse/Commands/ManagementCommands/SparkPool/NewAzureSynapseSparkPool.cs [hunk 2]
```diff
@@ -178,13 +178,13 @@ public override void ExecuteCmdlet()
...
                 {
                     Enabled = this.enableAutoScale,
                     MinNodeCount = AutoScaleMinNodeCount,
                     MaxNodeCount = AutoScaleMaxNodeCount
                 },
-                AutoPause = !EnableAutoPause ? null : new AutoPauseProperties
+                AutoPause = !EnableAutoPause ? new AutoPauseProperties { Enabled = EnableAutoPause.IsPresent } : new AutoPauseProperties
```
Comment (idear1203) [inline src/Synapse/Synapse/Commands/ManagementCommands/SparkPool/NewAzureSynapseSparkPool.cs:187]: >EnableAutoPause.IsPresent [](http://example.com/codeflow?start=83&length=25)

Just use `false` for simplicity

**Review by idear1203 — APPROVED** [review]
:shipit:

**Comment by VeryEarly** [comment]
please add changelog

**Review by idear1203 — APPROVED** [review]

**Comment by yanjungao718** [comment]
hi @idear1203 @VeryEarly , merging is blocked due to unauthorized, could you please help to do this?

**Pull request merged**